packages/graph/hash_graph/tests/rest-test.http

```diff
@@ -557,10 +557,10 @@ Content-Type: application/json
     ]
   },
   "graphResolveDepths": {
-    "dataTypeResolveDepth": 0,
-    "propertyTypeResolveDepth": 0,
-    "entityTypeResolveDepth": 0,
-    "entityResolveDepth": 0
+    "dataTypeResolveDepth": 10,
+    "propertyTypeResolveDepth": 10,
+    "entityTypeResolveDepth": 10,
+    "entityResolveDepth": 10
```

Comment:
Are these changes intentional?

Reply:
Previously, this test would have failed without this fix. Another former PR fixed this exact test but being a little bit more conservative shouldn't hurt here 🙂

Reply:
Cool! :)